cpmpy/tools/dimacs.py

```diff
@@ -30,9 +30,11 @@ def write_dimacs(model, fname=None):
         Writes CPMpy model to DIMACS format
         Uses the "to_cnf" transformation from CPMpy
 
-        # TODO: implement pseudoboolean constraints in to_cnf
+        .. todo::
+            TODO: implement pseudoboolean constraints in to_cnf
+
         :param model: a CPMpy model
```

Comment:
I see what you mean with 2 different comment styles.

I think this style is the exception, most code uses the other one and we should adopt that one for new code and when updating code